package.json

```diff
@@ -51,44 +51,49 @@
     "check-dev": "npm run lint && npm run test-dev"
   },
   "dependencies": {
-    "babel": "^5.5.8",
-    "babel-core": "^5.5.8",
+    "babel": "^5.8.21",
+    "babel-core": "^5.8.22",
```

Comment:
Some of the updates aren't necessary because the constraint with `^` matches what was already there, but I figured anything based off boilerplate should at *least* be the boilerplate version.